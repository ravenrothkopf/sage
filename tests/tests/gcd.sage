def int gcd(int a, int b) {
  while (a != b) {
    if (b < a) a = a - b
    else b = b - a
  }
  return a
}

def int main() {
    int x = 2
    int y = 14
    int a = 18
    int b = 9
    print(gcd(3, 15))
    print(gcd(x, y))
    print(gcd(99, 121))
    print(gcd(a, b))
}
