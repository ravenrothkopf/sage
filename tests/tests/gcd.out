3
2
11
9
